src/hooks/useHtmlRemover.ts

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect } from "react";
+
+export function useHtmlRemover(
```

Comment:
stubled upon the naming... was first expecting something like a sanitzeFunction ... maybe it could be something reflecting that some ui-elements are removed... or interaction-options... etc...
No need to change - just a hint.

Reply:
I've renamed it to useTldrawUiSanitizer to better reflect what it actually does, thanks for the suggestion.